### PR TITLE
Adding a file to new_branch

### PR DESCRIPTION
**Pull request by mugdhadalal**

### CODE REVIEW

File: calculator.py [hunk 1]
```diff
@@ -3,11 +3,27 @@
...
+  def div(self, x, y):
+    return x / y
 
   def multiply(self, x, y):
     return x * y
+ 
+  def add(self, x, y):
+    return x + y 
+
+  def mod(self, x, y):
```
Comment (ganeshhubale) [inline calculator.py]: Why do we need two ```add``` functions?

**Review by ganeshhubale — CHANGES_REQUESTED** [review]
Resolve conflict

File: calculator.py (hunk 1 above)
Comment (mugdhadalal) [reply calculator.py]: Basically, to perform program as many times as user wants we defined function.add is a function name.Function add will take two integers as input and return the sum.To perform math operation we used + symbol.

Comment (ganeshhubale) [reply calculator.py]: Hey @mugdhadalal Please read question carefully. I agree with your explaination but if you see you have defined ```add``` function for two times(line 17 and line 5)
- can't we remove one?

**Pull request merged**